cinema/models.py

```diff
@@ -41,10 +45,19 @@ class Movie(models.Model):
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to="create_custom_path")
```

Comment:
The `upload_to` parameter for the `image` field in the `Movie` model should be set to the `create_custom_path` method, not the string "create_custom_path". Change `upload_to="create_custom_path"` to `upload_to=Movie.create_custom_path`.